### ISSUE TITLE
Archive this repo, and just make all future issues and pull requests directly to the `JuliaLang/julia` repo

### ISSUE THREAD

**Issue by DilumAluthge** [JuliaLang/TOML.jl#36]
Julia 1.6 will be the new LTS. In Julia 1.6 and later, `TOML` is a standard library. Therefore, I don't think that we need to keep maintaining this repository as an external package.

@KristofferC What do you think about:
1. Archiving this repository.
2. All future issues and pull requests will be made directly to the `JuliaLang/julia` repo. In other words, the `TOML` stdlib (https://github.com/JuliaLang/julia/tree/master/stdlib/TOML) will be developed just like any other stdlib that lives inside the  `JuliaLang/julia` repo.

**Comment by simonbyrne** [comment]
Aren't we moving stdlibs out to their own repos?

**Comment by DilumAluthge** [comment]
> Aren't we moving stdlibs out to their own repos?

In almost every other case, yes.

The `TOML` stdlib is a very special case because the TOML parser is actually part of Base (because it is used in code loading).

**Comment by DilumAluthge** [comment]
But yes, for pretty much every other stdlib other than `TOML`, we would eventually like to move them out into their own external repos.

**Comment by KristofferC** [comment]
Yes, there are still some things to bring over from here though.

**Comment by DilumAluthge** [comment]
> Yes, there are still some things to bring over from here though.

Let's go ahead and do that, and then archive this repo.

I can make a PR to bring any last changes from this repo over to the JuliaLang/julia repo.

**Comment by ExpandingMan** [comment]
Considering the above, I think it would be nice if the TOML.jl that is part of Julia were re-structured into a low-level library that exposes a lot more internal functionality.

There is a lot more that it would be nice to do with TOML which this package cannot currently do and might not be appropriate for a core dependency of Julia.  In particular, as far as I know there isn't any functionality here for dealing with TOML entropy, that is writing has a fixed format and it is not possible to explicitly declare how the TOML contents should be written, and syntactic formatting information is mostly lost on reading.
